docs/content/how-tos/rules/FL0083.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FL0083
+category: how-to
+hide_menu: true
+---
+
+# UnneededRecKeyword (FL0083)
+
+*Introduced in `0.23.1`*
+
+## Cause
+
+Recursive functions (functions with a "rec" keyword) does not call themselves.
```

Comment:
Grammatical error. Either use plural and "do", or use singular ("function", "itself "not "function", "themselves")

Reply:
Nevermind, it got fixed since I started a review

Reply:
Yep thanks, I added a commit addressing this.